Equilobe.DailyReport.Models/Interfaces/IAtlassianWorklogDataService.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Equilobe.DailyReport.Models.Dashboard;
+using Equilobe.DailyReport.Models.Storage;
+using System;
+using System.Collections.Generic;
+
+namespace Equilobe.DailyReport.Models.Interfaces
+{
+    public interface IAtlassianWorklogDataService : IService
+    {
+        Dictionary<long, List<AtlassianWorklog>> GetAtlassianWorklogsByUserIds(long instanceId, List<long> users);
```

Comment:
maybe you can create a class to avoid using dictionaries.
```
InstanceWorklogs
{
   id;
   List<Worklogs>
}
```

It makes the code more readable

Reply:
It makes it more readable, but it's not as fast as a dictionary. I prefer using dictionary for fast manipulation. Adding and retrieving from a dictionary is O(1), while with a class, I have to search for the Id that I want, and doing that for each worklog, within each user might not be a good idea.

Reply:
How many items will be in the dictionary? 100 users? 1000 users? for these numbers it makes almost no difference.
you are doing heavy IO operations (access to JIRA server, access to current database), so these optimizations really don't make any differences